src/main/routing/urlBuilders.ts

```diff
@@ -35,3 +37,5 @@ export const buildUpdatesUrl = (journeyLocation: string) => {
 export const buildAboutUrl = (journeyLocation: string | undefined) => {
   return journeyLocation && `${journeyLocation}/about`;
 };
+
+export const buildVCProfileSettingsUrl = (nameID: string) => `/vc/${nameID}/settings/profile`;
```

Comment:
instead of creating this new one I would try to use `buildSettingsUrl` which is also concatenating stuff and we may want to change it in the future, but it takes the original vc's profile url and adds `/settings` which is what we are doing for other entities